src/js/modules/SectionHeader.js

```diff
@@ -1,26 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import ArrowRight from 'js/svg/ArrowRight';
 
-
-const SectionHeader = ({ isSerif, hasHighlight, hasArrow, children }) => (
+const SectionHeader = ({ isSerif, hasHighlight, children }) => (
   <h2 className={`coa_SectionHeader
     ${hasHighlight ? 'coa_SectionHeader--hasHighlight' : ''}
     ${isSerif ? 'coa_SectionHeader--isSerif' : ''} `
   }>
     {children}
-    {hasArrow && (
```

Comment:
I moved the arrow svg out, so I could nest it within a link that's passed in as children.